crates/common/src/model/world_state.rs

```diff
@@ -109,6 +111,10 @@ impl WorldStateError {
             Self::DuplicatePropertyDefinition(o, p) => {
                 E_INVARG.with_msg(|| format!("Duplicate property definition: {}.{}", o, p))
             }
+            Self::ChparentPropertyNameConflict(o, new_parent, prop) => E_INVARG.msg(format!(
```

Comment:
Is there a special reason the other branches use a closure here? Is it just that `.msg` didn't take `ToString` before?

Reply:
... Also now that I'm looking at this: we duplicate the error message between the `#[error(...)]` attributes and down here.

Reply:
"Is there a special reason the other branches use a closure here? "

Looks like user error. Problem between chair and keyboard. My bad. Go ahead and fix

Reply:
Days since I wished for stacked PRs: 0